src/components/DatePicker/Calendar.vue

```diff
@@ -105,17 +106,27 @@ export default class Calendar extends Vue {
     }
   }
 
+  // Emitted values should always been days at midnight (UTC)
   onInput(value: Date | DateRange | undefined): void {
-    if (Boolean(value) && !(value instanceof Date)) {
-      this.$emit('input', { ...value, duration: 'day' });
-    } else {
+    if (value == null) {
       this.$emit('input', value);
+    } else if (value instanceof Date) {
+      this.$emit('input', setDateToMidnight(value));
+    } else {
+      const newDateRange: DateRange = { ...value, duration: 'day' };
+      if (newDateRange.start) {
+        newDateRange.start = setDateToMidnight(newDateRange.start);
+      }
+      if (newDateRange.end) {
+        newDateRange.end = setDateToMidnight(newDateRange.end);
+      }
+      this.$emit('input', newDateRange);
     }
   }
```

Comment:
C'est pas déjà le cas, on a un paramètre UTC sur le composant donc il nous retourne toujours des heures a minuit par défault due a v-calendar non ?

Reply:
Hm... good point
When testing this on the playground yesterday, we saw dates not at midnight but at the time we were testing.
Maybe we've missed setting a prop

Reply:
Indeed! It seems we should use `timeAdjust` to do this (setting the timezone is not enough).

See https://vcalendar.io/datepicker.html#time-adjust

I'll do this, it's easier than recoding it ourselves!

Reply:
2cd56bb